code/modules/power/supermatter/supermatter.dm

```diff
@@ -664,6 +675,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 			if(combined_gas > MOLE_PENALTY_THRESHOLD)
 				radio.talk_into(src, "Warning: Critical coolant mass reached.", engineering_channel)
+				if(power > CRITICAL_POWER_PENALTY_THRESHOLD)
+					radio.talk_into(src, "Warning: Anti-mass spectrometer showing a five percent gain", engineering_channel)
```

Comment:
period

Reply:
ATH please learn how to use code suggestions
```suggestion
					radio.talk_into(src, "Warning: Anti-mass spectrometer showing a five percent gain.", engineering_channel)
```

Reply:
Can do